oidc_apis/utils.py

```diff
@@ -70,6 +70,9 @@ def additional_tunnistamo_id_token_claims(dic, user, token, request, **kwargs):
     if not tunnistamo_session:
         return dic
 
+    # Add Tunnistamo Session id as the "sid" (Session ID) claim
+    dic['sid'] = str(tunnistamo_session.id)
```

Comment:
I don't see the comment bringing much value, as it pretty much repeats what the code says quite clearly already.

Reply:
It's keeping up with the other comments. Also I think it's good to say what the "sid" means.

Reply:
Hmm, that's true. All the hard-to-understand claim names seem to have been documented there. Since that's the chosen path, then it's good to keep it that way.